src/main/java/com/minecolonies/coremod/colony/CitizenDataView.java

```diff
@@ -259,5 +295,36 @@ public void deserialize(@NotNull final ByteBuf buf)
         saturation = buf.readDouble();
 
         job = ByteBufUtils.readUTF8String(buf);
+
+        final int length = buf.readInt();
+        for (int i = 0; i < length; i++)
+        {
+            final String textComp = ByteBufUtils.readUTF8String(buf);
+            final TextComponentTranslation textComponent = new TextComponentTranslation(textComp);
+            latestStatus[i] = textComponent;
+        }
+
+        colonyId = buf.readInt();
+
+        NBTTagCompound compound = ByteBufUtils.readTag(buf);
+        inventory = new InventoryCitizen(this.name, true);
+        final NBTTagList nbttaglist = compound.getTagList("inventory", 10);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "nbttaglist". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "nbttaglist" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1481)